src/Options.php

```diff
@@ -179,23 +179,23 @@ public function setMbDetectOrder($detectOrder): void
     }
 
     /**
-     * Gets whether the stacktrace must be auto-filled.
+     * Gets whether the stacktrace will be attached on captureMessage.
      *
      * @return bool
      */
-    public function getAutoLogStacks(): bool
+    public function getAttachStacktrace(): bool
     {
-        return $this->options['auto_log_stacks'];
+        return $this->options['attach_stacktrace'];
     }
 
     /**
-     * Sets whether the stacktrace must be auto-filled.
+     * Sets whether the stacktrace will be attached on captureMessage.
      *
-     * @param bool $enable Flag indicating if the stacktrace must be auto-filled
+     * @param bool $enable flag indicating if the stacktrace will be attached to captureMessage calls
```

Comment:
Not that important after all, but the description of the parameter should always start with a capitalized letter